ys@0.1.87.rb

```diff
@@ -0,0 +1,52 @@
+class YsAT0187 < Formula
+  desc "Program in YAML — Code is Data"
```

Comment:
Can we use single quotes where interpolation is not used?

Reply:
> Can we use single quotes where interpolation is not used?

@ingydotnet In this case no, as this is enforced by the Home`brew style` command, which is build on top of their opinionated _[Rubocop](https://rubocop.org)_ config… It actually used to enforce single quotes, but they switched it to double a while back.